### DISCUSSION TITLE
Octernship confusion

### DISCUSSION THREAD

**Discussion by ishaan812** [community/community#50296] [GitHub Education]
Hey, I'm not really sure how to submit the task and ask for a review for an Octernship Task. The flow the organisation follows is:
![image](https://user-images.githubusercontent.com/70190533/225842158-a4af5d74-f9bf-4c68-8135-57170ac094e1.png)
I think I made a mistake and accepted the Pull Request, but that was because I couldnt see how to ask for a review on the Pull Request since there was no reviewer being shown. The pull request also mentioned that the reviewer would be able to see the changes on the Feedback branch itself. So i ended up merging the changes. I'm just confused how I can ask for a review now since I have already merged the auto generated Pull request by Github Classroom.

Please help out, I spent a bunch of time on the task which I would not want to go to waste :/

**Comment by AbhijithGanesh** [comment]
I've requested for a review from one of the maintainers, is that OK ?

**Comment by dennismuchene** [comment]
> I've requested for a review from one of the maintainers, is that OK?
